config/initializers/redis.rb

```diff
@@ -9,14 +9,16 @@
   ENV['REDIS_URL'] = "redis://#{password.blank? ? '' : ":#{password}@"}#{host}:#{port}/#{db}"
 end
 
-Redis.current = Redis.new(
+namespace = ENV.fetch('REDIS_NAMESPACE') { '' }
```

Comment:
confirmed! without setting a namespace all keys have just : prepended which is not the desired outcome

i will add a lil check to see if we have a namespace set and if not,  avoid using the namespacing features entirely

good catch! <3